db/schema.rb

```diff
@@ -125,6 +125,13 @@
     t.datetime "updated_at",                          null: false
   end
 
+  create_table "products", force: :cascade do |t|
```

Comment:
Plus I think git ignore will be good because sometimes if someone branches off for a while, they might have a slightly different schema by the time they're branching back in.  Lets add the schema.rb to git ignore and then close and merge this.